pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/PersistentTopics.java

```diff
@@ -270,6 +270,30 @@ public void updatePartitionedTopic(
         internalUpdatePartitionedTopic(numPartitions, updateLocalTopicOnly);
     }
 
+
+    @POST
+    @Path("/{tenant}/{namespace}/{topic}/createMissedPartitions")
+    @ApiOperation(value = "Create missed partitions of an existing partitioned topic.", notes = "It only try to create missed partitions of existing non-global partitioned-topic")
```

Comment:
```suggestion
    @ApiOperation(value = "Create missed partitions of an existing partitioned topic.", notes = "It only tries to create missed partitions of existing non-global partitioned-topic")
```

Reply:
Also it might be worth pointing out that this call is a best-effort operation. If it fails to create a partition, it doesn't throw any exceptions.